components/hab/static/template_gitignore

```diff
@@ -0,0 +1,2 @@
+.DS_Store
+results/
```

Comment:
This is a good question and one I don't have a great answer for myself. I think if I were working that way, it might bug me a bit to have my tree noised up with a bunch of .gitignores -- especially if I've already put one (as we have) at the top level -- but I'd probably just tidy things up manually once I noticed what was happening. But that's not the awesomest user experience, I agree. 

We could maybe locate the first .git ancestor, look for a .gitignore there, and create/append at that location instead -- but would that always be the right place? I'm not sure. 🤔